src/test/java/net/openhft/chronicle/queue/bench/QueueMultiThreadedJLBHBenchmark.java

```diff
@@ -111,7 +115,7 @@ public void init(JLBH jlbh) {
                 while (!stopped) {
                     long beforeReadNs = System.nanoTime();
                     try (DocumentContext dc = tailer.readingDocument()) {
-                        if (dc.wire() == null)
+                        if (!dc.isPresent())
```

Comment:
`dc.wire() == null` is not a valid test when using the ring buffer

Reply:
I think the `wire == null` pattern is used all over the place. Should we update all of them (code, demos, docs etc) to `isPresent()`?

Reply:
+1 @minborg nice idea